src/pilot/assign_role.py

```diff
@@ -381,18 +385,39 @@ def get_raid_controller_physical_disk_ids(drac_client, raid_controller_fqdd):
         key=physical_disk_id_to_key)
 
 
-def define_storage_logical_disks(drac_client, raid_controller_name):
+def define_storage_logical_disks(drac_client, raid_controller_ids):
+    boss_controller = None
+    raid_controller = None
```

Comment:
It would be great if we could make this handle the case where there are multiple RAID controllers.  To do this we could rename this variable to raid_controllers and set it equal to the empty list

Reply:
I have updated code to support multiple controllers.